druntime/src/core/internal/gc/impl/conservative/gc.d

```diff
@@ -160,7 +160,7 @@ class ConservativeGC : GC
 
     Gcx *gcx;                   // implementation
 
-    static gcLock = shared(AlignedSpinLock)(SpinLock.Contention.lengthy);
+    static gcLock = shared(AlignedSpinLock)(SpinLock.Contention.brief);
```

Comment:
``brief`` is zero here.

``k`` Will always be 0, therefore no yielding is possible.

I'm pretty certain ``lengthy`` would be the right value here due to possible syscalls.

Reply:
immutable step = 1 << contention;
        
        With contention set to brief, step becomes 1. So the thread still yields, but it does not yield that often.

Reply:
Oh right yes.